### ISSUE TITLE
flowjo_to_gatingset ignore range as defined in FCS files

### ISSUE THREAD

**Issue by Close-your-eyes** [RGLab/CytoML#155]
Hello.

Is there a way to tell flowjo_to_gatingset that is should ignore the range keyword from FCS files? Similar to the truncate_max_range  argument in flowCore::read.FCS? 

I found a workaround by modifying the respective keywords in the FCS files. However, this seems to be generic problem / convenient feature.

Thanks.

**Comment by mikejiang** [comment]
are you referring to $PnR keywords? They are required according to the standard, Why would you want to ignore them?

**Comment by Close-your-eyes** [comment]
Yes I do mean these keywords. In my case it appears that the machine actually has resolution for higher values in fsc and ssc channels.

**Comment by mikejiang** [comment]
These keywords are used extensively in cytoverse tools, thus it is important to make sure they are accurate and consistent, which was what you already did
